scripts/kvm-test.py

```diff
@@ -193,6 +190,16 @@ def load_config(self):
                     help='boot test image')
 
 
+cc_group = parser.add_mutually_exclusive_group()
+cc_group.add_argument('-f', '--autoinstall-file', action='store',
```

Comment:
My hunch is that we should remove this argument entirely.
Most of the time, if sending autoinstall data, it's interesting to have autoinstall on the kernel command line.
If we did a quick yaml load of the cloud-config and checked for top-level key 'autoinstall', we could switch over to only using --cloud-config-file and send the autoinstall kernel command line (or not) based on autoinstall being in that file.

Also, I wanted to remove the `-f` short option, which somehow didn't get discussed during that kvm-test discussion months ago (or at least didn't make my notes).